spec/paperclip/io_adapters/registry_spec.rb

```diff
@@ -18,7 +18,7 @@ def initialize(target); end
   context "registered?" do
     before do
       class AdapterTest
-        def initialize(target); end
+        def initialize(target, _ = {}); end
```

Comment:
Unused method argument - `target`. If it's necessary, use `_` or `_target` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.